content/android-privacy-policy.md

```diff
@@ -1,14 +1,20 @@
 # Syncthing-Android Privacy Policy
 
-The app syncthing-android does not collect any user information.
+Read this document to understand how the app Syncthing-Android uses data.
+
+---
+
+Syncthing-Android does not collect any user information.
 
 The app does not store any location data.
-It requires the permission to access the location in the background, to look up the currently active Wi-Fi's SSID.
-That information is used to enable syncing only on user-configurable Wi-Fi's.
-The app never actually looks up your location in the process, and thus doesn't use or store location data.
+The location permission is used only to access background location in order to check for an SSID of the currently active WiFi.
```

Comment:
What's "background location". And we don't access location, so why say we do. Also the permissions "location in the background" is an entity, a phrase used by Google - having that verbatim is more important than nice sentences :)

Reply:
I took the phrase directly from Google… See https://android-developers.googleblog.com/2020/02/safer-location-access.html ("Preventing unnecessary access to background location").